vocab/credentials/v2/vocabulary.yml

```diff
@@ -33,6 +33,11 @@ class:
     label: Refresh service
     comment: A Refresh Service is a mechanism that can be utilized by software agents to retrieve an updated copy of a Verifiable Credential.
 
+  - id: Credential
+    label: Credential
+    comment: |
+      A Credential is a set of one or more claims made by an issuer.
```

Comment:
Not specified as part of this vocabulary and, by default, yes, it can be a `proof`.

Never forget: a `VerifiableCredential` is also a `Credential` (that is what subclassing means).

Reply:
I don't see the subclass as defining any new information then... and it seems... not helpful.

It leads me to consider that both names are equally useful, and as such, we need only define 1.